src/users/__tests__/usersReducer-test.js

```diff
@@ -51,22 +52,15 @@ describe('usersReducer', () => {
 
     /**
      * Check that an update event with supplied `person` works.
-     *
-     * May omit `user_id` to avoid repetition.
      */
     // Tell ESLint to recognize `check` as a helper function that runs
     // assertions.
     /* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expect", "check"] }] */
-    const check = (personMaybeWithoutId: $Rest<User, { ... }>) => {
-      const person = {
-        user_id: theUser.user_id,
-        ...personMaybeWithoutId,
-      };
+    // TODO: Try to make `user_id` optional again.
+    const check = (person: $PropertyType<RealmUserUpdateEvent, 'person'>) => {
```

Comment:
I have a fix for this, which I showed you in the office. Here's [a try-flow link](https://flow.org/try/#0C4TwDgpgBAqlC8AoKAfKBvKAPAXFAzsAE4CWAdgOYA0UARnmQK4C2tERUAvsmprgcXLUoAYwYs2HTgG5EiUJCgB5BBigBqRnjgy5Aej1QAKgAsS+KAHcA9kQDW+HIgAUW2AEoEAPgzJR1skJsbVVMADoIxi5ZbkQDOkZgKxMIMigSYAByCzYzMgATKABDNIASAFEAGwhmVOAjcGgAWibkkhETAhNrRkrCtigIAEdGEgA3IuqyYCcFCHxykfHJuoAeCura6YbIVaUaTMZMrxoYL2d3WXiWqGAUoghs4rT2IlsnVzwNmrqdiD2DkcvJ54D50H4RAEgvxvlt6o0AVBDsdQlAImEorpYvFTNBXrYoCYimBIIECJARCQAGbtSaVEDJVK3FJQWG-RrpCzkOKGO7QO5CKyPB7kh5FfJCD5uNnbBH7JFAkFgiFQpL8ODwNTozExfSGEqFfLWeZkTJJIkkpmWFJpDJPAL09JpPnMwWWYXQKmVayWIQ85nQfBgMWFcjAaxS7RK3xQfyBNVfKo-WW7eXInya8KRaKIbGGACCVlsdiKb0YBU5t2sUGYRTs0CKtw5YFLRVqwHYzKKWRyPQKEH6IH9Lpl8MgNANUEY+H5RKS5EIEHFUGsVIDrKTcL+6WAM8qVKcqxgic27NTgOO5zcMGj4NjkPjwVgqO1ObzUGcAGUaiVgLTKo6GQEBANQWJUJD1l2ST4N0vT9A2LxEG8HDUgG-r4G2s6CtOgourQfaFHMUASg8IjAIBFglIMSG2GEsYAEKJP69wQDQYbsGA1j4Lhc5NooLZEJhHYcDOoFVl0JAQH0aHxEGIZQFSbzMMk3YQGMna0IwFCVkUVJUhAZGCqOfxhO4cjyI0CxLBMUzAOsAAKbyQEQoB-KsmCMAA-CEnAXicz5oGM1gkPk5yXBZkBWaMNlrKUjnWM5rkIh5Pl+acYWyHMUXLLZ6ybme-wed5z6+QqxynKgUBBSFGURfMizRSs0x5aeKaFVOqVlf5ZwXLIiAQFgnEuQp5aGQEfH1dZTV2UYpx4LNUAAPoAJLTOwZCTAAItYABy1jAPZRT4I4z6akYYUYLE-oPkEHQGXYqjOP6sbOfgASLSeyZjv8ABKS6VMwMAzkQMBgPkqnlOp0wHK9ATlf6t7PXGQSw2Qn1bgif2TIDwOg+DHaQ3UMPsG9poZr48SxrG6Ko4tVBI7o8S6MOHL2STAQAMLWMwzDjUglOVZ+jC0DOB1Oewf7zEewNKEQ9H7TQ6BoJowOLSFeDMCQWADjQ6j4cAi0+mQ7Bq-kGtawOqCcF4SNoELIsQGLCUS5J+DS+wsvy8AivK9OJvqzWFv5Lr+RSeM7AgItP4kJU5va4UKDW7bGAoEjVMaH7RCm+IrDsPTAvp+oIjTuGzCLcGq4xxAi00lJZsCKQlD54Y6eJ8nSsZ6rAdMLnRC68blhR7WMd4IQjfaYnfXxANQ1JERWMA0D7B4xDUNJJmqcC+oCV4IcYOqZkzdU+oqN4GzRCk1zPMBP6k-mdPg22HPHJL0Qnv7aor9XIYM9PxNsDA1CJvFund-b1x7pII+sZ1Ch3AupIgkdo6VGKmPIQUCND60NpYY2WcQrFQgZ2NATAAJHzvnqQYj9hpEXtqLeKiVXarAWmcVQpQlC0AAFYAFliQkCPDQVYABpGgAA1c4AjozGQRAtARXgvCyCAA) for the exploration we did of this Flow issue.